app/models/invitation.rb

```diff
@@ -0,0 +1,14 @@
+class Invitation < ActiveRecord::Base
+  def self.find_by_slug(slug)
+    Invitation
+      .where(slug: slug)
+      .where(Invitation.arel_table[:expiration_date].gt(Date.today))
+      .take!
+  rescue RangeError
+    raise RecordNotFound.new('The invitation does not exist or it has expired')
```

Comment:
:warning: Dates do not take into consideration Timezones nor hours or minutes.  Use `Time.now` or `Time.zone.now`